### PR TITLE
Merge content of stream graphs.md

### PR DESCRIPTION
**Pull request by naferx**
Ref #23052

### CODE REVIEW

**Comment by naferx** [comment]
@raboof  This PR is ready for review

File: akka-docs/src/main/paradox/scala/stream/stream-graphs.md [hunk 1]
```diff
@@ -44,10 +44,14 @@ the below hand drawn graph into Akka Streams:
 
 Such graph is simple to translate to the Graph DSL since each linear element corresponds to a `Flow`,
 and each circle corresponds to either a `Junction` or a `Source` or `Sink` if it is beginning
-or ending a `Flow`. Junctions must always be created with defined type parameters, as otherwise the `Nothing` type
-will be inferred.
+or ending a `Flow`. @scala[Junctions must always be created with defined type parameters, as otherwise the `Nothing` type
```
Comment (raboof) [inline akka-docs/src/main/paradox/scala/stream/stream-graphs.md]: Perhaps remove the `@java[ ]` and move the `.` inside the `@scala[...]`?

**Review by raboof — CHANGES_REQUESTED** [review]
Looks pretty good, I noticed a couple of 'empty' `@java[ ]` directives, these can be removed entirely. Almost ready to go!

File: akka-docs/src/main/paradox/scala/stream/stream-graphs.md (hunk 1 above)
Comment (naferx) [inline akka-docs/src/main/paradox/scala/stream/stream-graphs.md]: Done

**Review by raboof — APPROVED** [review]
Looks good thanks!

**Pull request merged**